### DISCUSSION TITLE
Error while parsing theme

### DISCUSSION THREAD

**Discussion by PedroPepeu** [Senshi111/debian-hyprland-hyprdots#7] [Q&A]
Hello, I am new to hyprland and ricing, but I enjoyed your theme, and downloaded it today, but I am facing some problems with rasi, being specific, theme.rasi
`* {
    main-bg:            E6;
    main-fg:            ;
    main-br:            ;
    main-ex:            ;
    select-bg:          ;
    select-fg:          ;
}`

its almost blank, I don`t know if this is the problem, but it acuses that the location of the syntax error, is in the line 2 column 13 to line 2 column 27.

I happens every time I try to use something from rofi

**Comment by Senshi111** [comment]
You mean there's no background in rofi?

**Reply by PedroPepeu** [reply]
Thats what Iam not understanding, its working, there is background and I can change it, Also the keybinds work too, but I cant change the theme and open other things, the workbar does not appear two, I cant se my battery and nothing like it, facing some problems.
I also dont know if it is in common but above in my screen are showing that there is an config error at line 10 of userprefs.conf, thaks for the answer

**Reply by Senshi111** [reply]
What's on that line 10? Did you tried to commented out? Also make sure that all apps are installed from build folder. Try to install them one after another and see if it's failed on some place or build success. What debian version are you using?

**Reply by PedroPepeu** [reply]
Iam using debian 12, I`ve try to comment, but appears another error,
in line 10 the error is about "force_zero_scalling = true"
and the another error is in line 7 "env = GDK_SCALE, 1" it show that has no such field for rgba(cc)rgba(cc)45deg, if I comment it, it still showing the error, I will try to isntall them one after another...

**Reply by Senshi111** [reply]
I don't know if hyprland build will work on debian 12. Are you sure that install hyprland script build  without error?

**Reply by PedroPepeu** [reply]
Probably not... I think I will download arch, it will be a headache but, why not...
Thank you for everything man!

**Reply by Senshi111** [reply]
I'm on fedora right now. I also have same thing for fedora if you want, you can try if you like more stable distro but also up to date
